### ISSUE TITLE
Electron rebuild failing strangely on Appveyor

### ISSUE THREAD

**Issue by MarshallOfSound** [electron/electron-rebuild#64]
Error
-----
```
Error: ENOENT: no such file or directory, open 'C:\projects\google-play-music-desktop-player-unofficial\node_modules\undefined'
```

For some reason it is attempted to rebuild a module with the name `undefined`.  It doesn't do this locally or on any other windows machine I have available.  Looking for any suggestions as to what is going on here.

**Node:** 4.4.x
**NPM:** 3.x.x

Full Stack Trace
---------------------

```
C:\projects\google-play-music-desktop-player-unofficial\node_modules>.bin\electron-rebuild.cmd -f 
 
gyp WARN install got an error, rolling back install
gyp ERR! install error 
gyp ERR! stack Error: ENOENT: no such file or directory, open 'C:\projects\google-play-music-desktop-player-unofficial\node_modules\undefined'
gyp ERR! stack     at Error (native)
gyp ERR! stack     at Object.fs.openSync (fs.js:549:18)
gyp ERR! stack     at Object.fs.readFileSync (fs.js:397:15)
gyp ERR! stack     at readCAFile (C:\projects\google-play-music-desktop-player-unofficial\node_modules\npm\node_modules\node-gyp\lib\install.js:466:15)
gyp ERR! stack     at download (C:\projects\google-play-music-desktop-player-unofficial\node_modules\npm\node_modules\node-gyp\lib\install.js:438:22)
gyp ERR! stack     at C:\projects\google-play-music-desktop-player-unofficial\node_modules\npm\node_modules\node-gyp\lib\install.js:185:19
gyp ERR! stack     at C:\projects\google-play-music-desktop-player-unofficial\node_modules\npm\node_modules\mkdirp\index.js:30:20
gyp ERR! stack     at FSReqWrap.oncomplete (fs.js:82:15)
gyp ERR! System Windows_NT 6.3.9600
gyp ERR! command "C:\\Program Files (x86)\\nodejs\\node.exe" "C:\\projects\\google-play-music-desktop-player-unofficial\\node_modules\\npm\\node_modules\\node-gyp\\bin\\node-gyp.js" "install" "--target=0.37.8" "--arch=ia32" "--dist-url=https://gh-contractor-zcbenz.s3.amazonaws.com/atom-shell/dist"
gyp ERR! cwd C:\projects\google-play-music-desktop-player-unofficial\node_modules
gyp ERR! node -v v4.4.1
gyp ERR! node-gyp -v v3.3.1
gyp ERR! not ok 
 
 
Process exited with code: 1
Error: Process exited with code: 1
    at ChildProcess.<anonymous> (C:\projects\google-play-music-desktop-player-unofficial\node_modules\electron-rebuild\lib\spawn.js:55:26)
    at emitTwo (events.js:87:13)
    at ChildProcess.emit (events.js:172:7)
    at maybeClose (internal/child_process.js:827:16)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:211:5)
 
Rebuild failed
```

**Comment by MarshallOfSound** [comment]
I'm going to leave this open as there is underlying issue somewhere but for what it's worth this issue was fixed by changing node versions from `4.4.X` to `6.1.0`

**Comment by eugirdor** [comment]
I get the same error.  Upgrading to 6.1.0 did not resolve it for me.  I worked around it by creating a 0 byte file in the directory named `undefined`.
